app/src/main/res/layout/fragment_home.xml

```diff
@@ -2,79 +2,83 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.constraintlayout.motion.widget.MotionLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
Did you mean to wrap this inside a coordinator layout?

Reply:
Yes, the bottom sheet behavior depends upon it.

Reply:
That’s what I thought as well, but I tested it without it and it seemed to work?

Reply:
Ah I see the behavior now. I will play around with it.